vNext/extensions/applicationinsights-react-js/src/ReactPlugin.ts

```diff
@@ -20,7 +19,7 @@ export default class ReactPlugin implements ITelemetryPlugin {
     public priority = 165;
```

Comment:
cab you change this to 180? Properties plugin is 170 and this plugin sends telemetry in its initialization steps.